### PR TITLE
Add BLM banner as announcement bar

### PR DESCRIPTION
**Pull request by TheoBr**
Styling copied from React's topnav CTA

image

### CODE REVIEW

**Comment by TheoBr** [comment]
Just signed the contributor's license :)

**Comment by smithki** [comment]
Why was this closed? @davidmccabe

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -16,9 +16,16 @@ module.exports = {
   organizationName: 'facebookexperimental', // Usually your GitHub org/user name.
   projectName: 'Recoil', // Usually your repo name.
   themeConfig: {
-     algolia: {
+    announcementBar: {
+      id: 'black_lives_matter',
+      content:
+        '<div class="blm-bar">Black Lives Matter. <a style="color: rgb(97, 218, 251)" target="_blank" href="https://support.eji.org/give/153413/#!/donation/checkout">Support the Equal Justice Innitiative</a></div>',
```
Comment (afturner) [inline docs/docusaurus.config.js:22]: Should be "initiative"

Comment (TheoBr) [reply docs/docusaurus.config.js:22]: Oh man I copy-pasted this I have no idea how that happened 😅  good catch!

**Comment by jdan** [comment]
Hi, I'd also like to see your reasoning here @davidmccabe.

**Comment by natedunn** [comment]
Noticed and bummed by this quick closing with no reasoning.

**Comment by jamiebuilds** [comment]
I was really excited by Recoil, I was readying a blog post about it last week and I had been promoting it on Twitter. I'm not going to publish that blog post, and I'm not going to be using Recoil.

**Comment by pavermakov** [comment]
@jamiebuilds I don't think that just because they have rejected the PR they don't support BLM. In fact, the banner itself is quite obtrusive. Dev.to has a much better one.

Please don't give up on recoil and your block post.

**Comment by alexeyraspopov** [comment]
I just wish we could hear something from
@davidmccabe on their reasoning

**Pull request closed without merging**